unicodethingy.py

```diff
@@ -14,10 +14,10 @@
 
 print(counter.most_common(150))
 
-arrows = { chr(i) + ' (' + str(i) + ') ' + unicodedata.name(chr(i),'')
+arrows = ['{0} 0x{1:4X} {2}'.format(chr(i), i, unicodedata.name(chr(i),''))
```

Comment:
you could even make the formatting string a constant.

Reply:
Nevar!